live/src/main/scala/org/dbpedia/extraction/live/LiveExtractionConfigLoader.scala

```diff
@@ -90,7 +90,7 @@ object LiveExtractionConfigLoader
     val lang = Language.apply(landCode)
     val articlesSource : Source =
       if (item.getXML.isEmpty)
-        WikiSource.fromPageIDs(List(item.getItemID), new URL(apiURL), lang)
+        WikiSource.fromRevisionIDs(List(item.getItemID), new URL(apiURL), lang)
```

Comment:
this will be a problem, I see that the title is also there
https://www.mediawiki.org/wiki/Manual:RCFeed

 I suggest we extend a bit the LiveQueueItem to add the revision id's, now you mix page ID's with revisions
and also add the title that we get from RCStream